trestle/core/generators.py

```diff
@@ -130,15 +130,19 @@ def generate_sample_model(
 
     model_dict = {}
     # this block is needed to avoid situations where an inbuilt is inside a list / dict.
+    # the only time dict ever appears is with include_all, which is handled specially
+    # the only type of collection possible after OSCAL 1.0.0 is list
     if safe_is_sub(model, OscalBaseModel):
         for field in model.__fields__:
+            if field == 'include_all':
```

Comment:
What about `include_all` in `Profile`? There it seems to not be dictionary?

Reply:
In the Oscal profile include_all is just a pass.  In our code if include_all is not None it is regarded as true - so it can be anything.  We give the value of an empty Dict when we generate a sample - but that is the only place it happens.  Many of the sample profiles have an empty dict for the value.  So this is just an odd situation until they make it boolean in a future oscal release - but it should all work fine.